src/app/scenario/scenario.component.html

```diff
@@ -29,10 +29,28 @@ <h3>Scenarios</h3>
 </div>
 <div class="clr-row">
   <div class="clr-col-4">
-    <clr-datagrid [clrDgSingleSelected]="unusedSelectedScenario" (clrDgSingleSelectedChange)="editScenario($event)" [clrDisableSelection]="!selectRbac">
+
+    <clr-dropdown>
+      <button class="dropdown-toggle" clrDropdownTrigger>
+        <span *ngIf="category==''">All</span>
+        <span *ngIf="category=='__none__'">No Category</span>
+        <span *ngIf="category !='' && category != '__none__' ">{{category}}</span>
+        <clr-icon shape="caret down"></clr-icon>
+      </button>
+      <clr-dropdown-menu clrPosition="top-left" *clrIfOpen>
```

Comment:
```suggestion
      <clr-dropdown-menu clrPosition="bottom-left" *clrIfOpen>
```

If the List of Categories has some more entries, the Dropdown List gets hidden behind the frame of the Browser Window